src/trezor/trezor.cpp

```diff
@@ -252,12 +254,15 @@ namespace Trezor {
             case MessageType_PinMatrixRequest: handleMatrixRequest(msg_in); return;
             case MessageType_ButtonRequest: handleButtonRequest(msg_in); return;
             case MessageType_PassphraseRequest: handlePassphrase(msg_in); return;
+            case MessageType_PassphraseStateRequest: handlePassphraseStateRequest(msg_in); return;
             case MessageType_Features: handleFeatures(msg_in); return;
             case MessageType_EthereumAddress: handleAddress(msg_in); return;
             case MessageType_EthereumTxRequest: handleTxRequest(msg_in); return;
         }
 
-        bail("Unknown msg response: " + QString::number(msg_in.id));
+        if (false) {
```

Comment:
was this a debug addition to prevent crashing? Can you revert back please so we don't forget. I'd like to crash if there's an unexpected device message (once we have it all figured of course).